src/powerquery-language-services/analysis/analysisBase.ts

```diff
@@ -19,6 +19,8 @@ import type {
 } from "../providers/commonTypes";
 import { CommonTypesUtils, Inspection } from "..";
 import { EmptyHover, EmptySignatureHelp } from "../commonTypes";
+import { findScopeItemByLiteral, findTheCreatorIdentifierOfOneScopeItem } from "../inspection/scope/scopeUtils";
+import { IdentifierContextKind, NodeKind } from "@microsoft/powerquery-parser/lib/powerquery-parser/language/ast/ast";
```

Comment:
We make Ast a special case for importing, where instead of `NodeKind` you'd reference `Ast.NodeKind`. Without this you might end up with a file with two dozen imports from the same file.